json-schema/schema.json

```diff
@@ -372,6 +371,86 @@
         }
       }
     },
+    "vector_dimension": {
+      "title": "Vector Dimension Object",
+      "type": "object",
+      "required": [
+        "type",
+        "bbox"
+      ],
+      "properties": {
+        "type": {
+          "type": "string",
+          "const": "vector"
+        },
+        "axes": {
+          "type": "array",
+          "uniqueItems": true,
+          "items": {
+            "type": "string",
+            "enum": [
+              "x",
+              "y",
+              "z"
+            ]
+          }
+        },
+        "description": {
+          "$ref": "#/definitions/description"
+        },
+        "bbox": {
+          "title": "Spatial extent",
+          "type": "array",
+          "oneOf": [
+            {
+              "minItems":4,
+              "maxItems":4
+            },
+            {
+              "minItems":6,
+              "maxItems":6
+            }
+          ],
+          "items": {
+            "type": "number"
+          }
+        },
+        "values": {
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "oneOf": [
+              {
+                "$ref": "https://geojson.org/schema/GeoJSON.json"
+              },
+              {
+                "description": "ID or WKT",
+                "type": "string"
+              }
+            ]
+          }
+        },
+        "geometry_types":  {
+          "type": "array",
+          "uniqueItems": true,
+          "items": {
+            "type": "string",
+            "enum": [
+              "Point",
+              "MultiPoint",
+              "LineString",
+              "MultiLineString",
+              "Polygon",
+              "MultiPolygon",
+              "GeometryCollection"
```

Comment:
In other places you used `Multi Point`, `Multi LineString`, `Multi Polygon`; I guess this one is the correct one (standards compliant)?

Reply:
Where are these other places? I can't find any of these used with a space in-between.

Reply:
https://github.com/Open-EO/openeo.org/pull/59

Reply:
Ah, okay, these ones here are used in the STAC extension (and GeoJSON etc) for M2M, the others are just textual descriptions for humans.